lua/hawtkeys/utils.lua

```diff
@@ -127,4 +128,14 @@ function M.reduceHome(path)
     return reduced
 end
 
+---@param lhs string
+---@return string
+function M.sanitise_modifier_keys(lhs)
+    --- to lower anything in < >
+    local santisedLhs = lhs:gsub("<[^>]*>", function(match)
+        return match:lower()
+    end)
```

Comment:
```suggestion
    local santisedLhs = lhs:gsub("<[^>]*>", string.lower)
```

Reply:
hmm or maybe we should just sanitize the mappings before storing them instead of in during the search?